src/main/pages/che-7/hands-on-guidance/proc_writing-a-meta-yaml-file-and-adding-it-to-a-plug-in-registry.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// publishing-a-vs-code-extension-into-the-che-plug-in-registry
+
+// This module can be included from assemblies using the following include statement:
+// include::<path>/proc_proc_writing-a-meta-yaml-file-and-adding-it-to-a-plug-in-registry.adoc[leveloffset=+1]
+
+// The file name and the ID are based on the module title. For example:
+// * file name: proc_doing-procedure-a.adoc
+// * ID: [id='proc_doing-procedure-a_{context}']
+// * Title: = Doing procedure A
+//
+// The ID is used as an anchor for linking to the module. Avoid changing
+// it after the module has been published to ensure existing links are not
+// broken.
+//
+// The `context` attribute enables module reuse. Every module's ID includes
+// {context}, which ensures that the module has a unique ID even if it is
+// reused multiple times in a guide.
+//
+// Start the title with a verb, such as Creating or Create. See also
+// _Wording of headings_ in _The IBM Style Guide_.
+[id="proc_writing-a-meta-yaml-file-and-adding-it-to-a-plug-in-registry_{context}"]
+= Writing a meta.yaml file and adding it to a plug-in registry
+
+The plug-in meta information is required to publish a VS Code extension in an Eclipse Che plug-in registry. This meta information is provided as a `meta.yaml` file. To create a `meta.yaml` file for your extension, see the following instructions.
+
+[Discrete]
+== Procedure
+
+. Create a `meta.yaml` file in the following plug-in registry directory:
+`<apiVersion>/plugins/<publisher>/<pluginName>/<pluginVersion>/`
++
+//NOTE: Support of two or more extensions is broken. See the link:https://github.com/eclipse/che/issues/13578[GitHub description for this issue] for further information.
+
+. Edit the `meta.yaml` and provide the necessary information. The configuration file must adhere to the following structure:
+
+[literal]
+--
+apiVersion: `v3`        <1>
+publisher: `myorg`      <2>
+name: `my-vscode-ext`	<3>
+version: `1.7.2`        <4>
+type:                   <5>
+displayName: 	        <6>
+title: 			<7>
+description: 		<8>
+icon: `https://www.eclipse.org/che/images/logo-eclipseche.svg`	<9>
+repository: 		<10>
+category: 		<11>
+spec:
```

Comment:
Can you please adjust the callout numbers (<1>, <2>,...) so that they appear in the same line? (The way they are listed right now, <9> is off. Tip: I use the space bar instead of tabs to get the alignment right.)